qt/aqt/sidebar.py

```diff
@@ -111,16 +114,37 @@ def rename_deck(self, item: "aqt.browser.SidebarItem") -> None:
         self.browser.maybeRefreshSidebar()
         self.mw.deckBrowser.refresh()
 
+    def remove_tag(self, item: "aqt.browser.SidebarItem") -> None:
+        self.browser.editor.saveNow(lambda: self._remove_tag(item))
+
+    def _remove_tag(self, item: "aqt.browser.SidebarItem") -> None:
+        old_name = item.full_name
+
+        def do_remove():
+            self.mw.col.backend.clear_tag(old_name)
+            self.col.tags.rename_tag(old_name, "")
+
```

Comment:
In the long run it might be nicer to have a backend method which does both these things in one go, as there's not much use in clearing a tag without altering any notes. But it will do for now :-)